### PR TITLE
[one-cmds] Add tests for one-quantize with directory input

### PR DESCRIPTION
**Pull request by jinevening**
This adds tests for one-quantize with directory input.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: #7398
Draft PR: #7401

### CODE REVIEW

File: compiler/one-cmds/tests/one-build_013.cfg [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+output_path=inception_v3.circle
+input_arrays=input
+input_shapes=1,299,299,3
+output_arrays=InceptionV3/Predictions/Reshape_1
+converter_version=v1
+
+[one-quantize]
+input_path=inception_v3.circle
+output_path=inception_v3.dir.quantized.circle
+input_data=raw_files
```
Comment (seanshpark) [inline compiler/one-cmds/tests/one-build_013.cfg:21]: Seems `raw_files` is a folder but doesn't seem to exist in this PR

Comment (jinevening) [reply compiler/one-cmds/tests/one-build_013.cfg:21]: raw_files is generated when `prepare_test_materials.sh` is called.

**Review by seanshpark — APPROVED** [review]
LGTM

**Comment by seanshpark** [comment]
Sorry for late merge... I got lost with this by other works...

**Pull request merged**